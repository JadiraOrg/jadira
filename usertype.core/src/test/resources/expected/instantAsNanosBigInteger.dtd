<!ELEMENT dataset (
    INSTANTASNANOSBIGINTEGER*)>

<!ELEMENT INSTANTASNANOSBIGINTEGER EMPTY>
<!ATTLIST INSTANTASNANOSBIGINTEGER
    ID CDATA #REQUIRED
    INSTANT CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

