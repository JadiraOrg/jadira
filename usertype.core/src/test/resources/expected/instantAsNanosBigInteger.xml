<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "instantAsNanosBigInteger.dtd">
<dataset>
  <INSTANTASNANOSBIGINTEGER ID="0" INSTANT="1077728625760000000" NAME="test_0"/>
  <INSTANTASNANOSBIGINTEGER ID="1" INSTANT="321581025000000000" NAME="test_1"/>
</dataset>